src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -25,7 +33,7 @@ private Event createEventFromEntity(Entity entity) {
         String name = entity.getString("name");
         String description = entity.getString("description");
         String location_name = entity.getString("location_name");
-        String date = entity.getTimestamp("date").toString();
+        String date = entity.getTimestamp("date").toString(); // change it to String?
```

Comment:
nit: This has been addressed in the other comments so I think we can remove this comment now.